app/globals.css

```diff
@@ -136,3 +136,26 @@ html.dark {
   transform: translateX(0px);
   transform: translateY(0px);
 }
+
+@media (max-width: 767px) {
+  .customScrollbar::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .customScrollbar::-webkit-scrollbar {
+    width: 15px;
+  }
+
+  .customScrollbar::-webkit-scrollbar-track {
+    background: var(--background);
+    border-radius: 5px;
+  }
+
+  .customScrollbar::-webkit-scrollbar-thumb {
+    background-color: #6025c0;
```

Comment:
What if the org. deploying leaderboard has a different primary color? #283

Reply:
okay, I will use the colour theme from the tailwind.config.ts file